pkg/util/startup/startup_test.go

```diff
@@ -85,18 +85,13 @@ func TestStartupFailure(t *testing.T) {
 func TestStartupFailureRandomRange(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	// This test takes 30s and so we don't want it to run in the "blocking path"
-	// of CI at all, and we also don't want to stress it in nightlies as part of
-	// a big package (where it will take a lot of time that could be spent running
-	// "faster" tests). In this package, it is the only test and so it's fine to
-	// run it under nightly (skipping race builds because with many nodes they are
-	// very resource intensive and tend to collapse).
+	// This test takes 30s+, so we don't want it to run in the "blocking path" of
+	// CI. We also skip race builds as the test uses multiple nodes, which can
+	// cause the test to grind to a halt and flake out.
 	skip.UnderRace(t, "6 nodes with replication is too slow for race")
-	skip.WithIssue(t, 9999999999, "nicktrav will have a fix shortly")
-	// TODO(nicktrav): re-enable only under nightlies once the fix is out.
-	//if !skip.NightlyStress() {
-	//	skip.IgnoreLint(t, "test takes 30s to run due to circuit breakers and timeouts")
-	//}
+	if !skip.Stress() {
```

Comment:
FYI: This needs to be `skip.NightlyStress()`, if you want it to be that this test is only run in the nightly stress runs. As-is, this test will only be executed when run locally via `dev test --stress`, which is probably not what you intended.